src/validate.js

```diff
@@ -30,24 +29,35 @@ var resolutionSchema = exports.resolutionSchema = Joi.alternatives().try(
 
 var browsersSchema = exports.browsersSchema = Joi.array().min(1).unique().items(
   Joi.object().keys({
-    browserName: Joi.string().valid(['chrome', 'firefox']).required(),
+    browserName: Joi.string().valid('chrome', 'firefox').required(),
     includeRules: includeRulesSchema,
     excludeRules: excludeRulesSchema
   }),
   Joi.object().keys({
-    browserName: Joi.string().valid(['chrome', 'firefox', 'safari', 'microsoftedge', 'internet explorer']).required(),
+    browserName: Joi.string().valid('chrome', 'firefox', 'safari', 'microsoftedge', 'internet explorer').required(),
     version: Joi.string().required(),
     includeRules: includeRulesSchema,
     excludeRules: excludeRulesSchema
   })
-);
+).when('sauce.launchSauceConnect', { is: true, then: Joi.array().min(1).unique().items(
+  Joi.object().keys({
+    browserName: Joi.string().valid('chrome', 'firefox').required(),
+  }).forbidden(),
+  Joi.object().keys({
+    browserName: Joi.string().valid('chrome', 'firefox', 'safari', 'microsoftedge', 'internet explorer').required(),
+    version: Joi.string().required(),
+    includeRules: includeRulesSchema,
+    excludeRules: excludeRulesSchema
+  }).required()
+).error(new Error('Can not run screener browsers when sauce connect tunnel is being used')) });
```

Comment:
The reason I move this block outside is because this version of `Joi` does not support reference parent, it only supports reference sibling and sibling's children.